bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/internal/p2/repository/Transport.java

```diff
@@ -28,21 +29,28 @@ public abstract class Transport {
 	 * monitor. If the <code>target</code> is an instance of {@link IStateful} the resulting status
 	 * is also set on the target. An IStateful target is updated with status even if this methods
 	 * throws {@link OperationCanceledException}.
-	 * 
+	 *
 	 * @returns IStatus, that is a {@link DownloadStatus} on success.
 	 * @param toDownload URI of file to download
 	 * @param target OutputStream where result is written
 	 * @param startPos the starting position of the download, or -1 for from start
 	 * @param monitor where progress should be reported
 	 * @throws OperationCanceledException if the operation was canceled.
```

Comment:
We need to comment in javadoc with `@deprecated <alternative>` describing that this method isn't used anymore and what other code should be used/implemented instead.

Reply:
done